src/plugins/ecs/ecs_compares/__init__.py

```diff
@@ -80,16 +100,62 @@ def compare_environment(team_env, master_env, jenkin_build_terms):
     3 - branch
     """""
     result = 0
+
     if jenkin_build_terms[0] in master_env or jenkin_build_terms[1] in master_env:
```

Comment:
I'll only say this once, instead of every time I see it in the code, but it's 'jenkins', not 'jenkin' :)